sysboundary/ionosphere.h

```diff
@@ -395,6 +394,10 @@ namespace SBC {
          const uint popID,
          const bool calculate_V_moments
       ) override;
+      virtual void setupL2OutflowAtRestart(
```

Comment:
Instead of having these warnings in all the "wrong" boundaries, I think it should exist in the base class as a warning and be just missing from all the others.